actions/generate-terraform-plan/action.yml

```diff
@@ -153,6 +157,24 @@ runs:
           fi
         done
 
+    - name: Determine Reviewer
+      if: inputs.enable_pr_creation == 'true' && github.ref_name == inputs.pr_creation_branch && steps.plan.outputs.has_changes == 'true'
+      id: reviewer
+      shell: bash
+      run: |
+        # Use fallback_reviewers only if github.actor is a bot or the event is scheduled
+        if [[ "${{ github.actor }}" == *"[bot]"* ]] || [[ "${{ github.event_name }}" == "schedule" ]]; then
+          if [[ -n "${{ inputs.fallback_reviewers }}" ]]; then
+            # Clean up comma-separated list by removing extra whitespace
+            CLEANED_REVIEWERS=$(echo "${{ inputs.fallback_reviewers }}" | sed 's/ *, */,/g' | sed 's/^ *//g' | sed 's/ *$//g')
```

Comment:
This is definitely safest, but github usernames can't have spaces in them right?  So probably just `sed 's/ //g'` would work.